apps/web/components/booking/BookingListItem.tsx

```diff
@@ -179,7 +178,7 @@ function BookingListItem(booking: BookingItemProps) {
           id: "reschedule",
           icon: "clock" as const,
           label: t("reschedule_booking"),
-          href: `${bookerUrl}/reschedule/${booking.uid}${
+          href: `${WEBAPP_URL}/reschedule/${booking.uid}${
```

Comment:
this probably won't work for organization booking urls

Reply:
because they have custom domains

Reply:
When the user visits `${WEBAPP_URL}/reschedule/...`, if the org booking url has custom domain, the user gets redirected to the appropriate org domain. See below Loom Link.

https://www.loom.com/share/06519395de4a498f9b822eb3ef64b04e?sid=5778459a-2995-441a-a794-c48128ff5598

Reply:
Nope. just tested on prod and the original code works fine here

Reply:
Yeah. That is true only for the users that have been migrated.


So, a brand new org user won't have this behaviour for his bookings.

Reply:
We would probably need to use `getBookerBaseUrl` that accepts organizationId and organizationId has to be derived from booking like this probably

https://github.com/calcom/cal.com/blob/c5b8ad89157beaceb7db65f7113f6e35fd2c3ef1/packages/features/bookings/lib/handleNewBooking.ts#L1612